### PR TITLE
Update feGaussianBlur IE, Chrome compatibility

### PR DESCRIPTION
**Pull request by zineeworld**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [x] Data: if you tested something, describe how you tested with details like browser and version
I tested it on IE9/10/11. feGaussianBlur works on IE 10 and on higher versions.
And it also works on Chrome, but the results were missing so i fixed it. Attached are the test result screenshots. https://github.com/mdn/browser-compat-data/issues/7006
- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any
https://github.com/mdn/browser-compat-data/issues/7006

### CODE REVIEW

**Comment by foolip** [comment]
Hi @zineeworld, I see that this is your first contribution to BCD, so allow me to say welcome!

I'll go ahead and review this now.

File: svg/elements/feGaussianBlur.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Element/feGaussianBlur",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```
Comment (foolip) [inline svg/elements/feGaussianBlur.json:9]: I've used http://staging-dot-mdn-bcd-collector.appspot.com/tests/api/SVGFEGaussianBlurElement to confirm support back to Chrome 15, so this is correct. It's also supported on current Chrome for Android, so I'll set that to true as well.

File: svg/elements/feGaussianBlur.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
             },
             "chrome_android": {
               "version_added": null
```
Comment (foolip) [inline svg/elements/feGaussianBlur.json:12]:
```suggestion
              "version_added": true
```

File: svg/elements/feGaussianBlur.json [hunk 3]
```diff
@@ -21,7 +21,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": "10"
```
Comment (foolip) [inline svg/elements/feGaussianBlur.json:24]: Testing https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feGaussianBlur#Example in IE 9 and 10 as you did in https://github.com/mdn/browser-compat-data/issues/7006 confirms this.

One wrinkle is that the element and its interface seems to be supported in IE 9 per http://staging-dot-mdn-bcd-collector.appspot.com/tests/api/SVGFEGaussianBlurElement, but that's not a useful level of support, so I agree with setting "10".

**Review by foolip — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Welcome @zineeworld!

**Pull request merged**